package.json

```diff
@@ -82,6 +82,7 @@
     "i18next": "^22.0.0 || ^23.0.0 || ^24.0.0 || ^25.0.0",
     "i18next-browser-languagedetector": "^7.0.0 || ^8.0.0",
     "i18next-http-backend": "^2.0.0 || ^3.0.0",
+    "react": ">= 16.8.0",
```

Comment:
Same as: https://github.com/i18next/react-i18next/blob/v15.5.2/package.json#L72C5-L72C26

`tsup` does not bundle packages that are listed in `peerDependencies`.

> By default tsup bundles all import-ed modules but dependencies and peerDependencies in your package.json are always excluded
> https://tsup.egoist.dev/#excluding-packages